modules/tiles/src/tileset/tile-3d.js

```diff
@@ -170,16 +170,32 @@ export default class TileHeader {
     }
   }
 
+  // If skipLevelOfDetail is off try to load child tiles as soon as possible so that their parent can refine sooner.
+  // Tiles are prioritized by screen space error.
   _getPriority() {
+    const traverser = this.tileset._traverser;
+    const {skipLevelOfDetail} = traverser.options;
+    const maySkipTile = this.refine === TILE_REFINEMENT.ADD || skipLevelOfDetail;
+
     // Check if any reason to abort
-    if (!this.isVisible) {
+    if (maySkipTile && !this.isVisible && this._visible !== null) {
       return -1;
     }
     if (this.contentState === TILE_CONTENT_STATE.UNLOADED) {
       return -1;
     }
 
-    return Math.max(1e7 - this._priority, 0) || 0;
+    const parent = this.parent;
+    const useParentScreenSpaceError =
```

Comment:
If you are able to add some more comments (given that you have spent time understanding the algorithm) that would be nice. Imagine the questions a new person would have when looking at this code.

Reply:
Added some comments.